lib/Doctrine/ORM/Tools/SchemaTool.php

```diff
@@ -141,7 +143,9 @@ private function processingNotRequired($class, array $processedClasses)
      *
      * @return Schema
      *
-     * @throws \Doctrine\ORM\ORMException
+     * @throws ORMException
+     * @throws DBALException
+     * @throws SchemaException
```

Comment:
`DBALException`and `SchemaException` are part of DBAL. Are those exceptions caught in ORM by using this method? If not, I would expect that an `ORMException` would be thrown if internally the DBAL exceptions occur.